src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -450,10 +452,10 @@ AthensSurfaceExamples class >> example7 [
 		
 			can pathTransform translateX: 50 Y: 50.
 			can drawShape: (0@0 corner: 50@50).
-
-			Display getCanvas drawImage: surf asForm at: 0@0
+			
 		].		
-	]
+	].
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?